app/models/send_sms.rb

```diff
@@ -0,0 +1,17 @@
+require 'twilio-ruby'
+
+class SendSMS
+  def self.send_text(phone_nuber)
+    account_sid = "ACb5d37ec881dbbc6e2b769dca8d7e2a33"
+    auth_token = ENV['AUTH_TOKEN']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.